### PR TITLE
Manage examples using lerna

### PR DESCRIPTION
**Pull request by sonalivdeshpande**
The aim of this PR is:
1. To manage all examples using lerna. The repo uses lerna to manage a subset of its packages, but not all. This makes understanding where to install/build for a given package unclear. 
[AB#1655](https://dev.azure.com/fluidframework/internal/_workitems/edit/1655)

2. Update dependabot to open prs for specific packages
[AB#2218](https://dev.azure.com/fluidframework/internal/_workitems/edit/2218)

3. Validate that all packages are managed by lerna in CI 
[AB#1654](https://dev.azure.com/fluidframework/internal/_workitems/edit/1654)

4. Ensure all examples are build in CI
[AB#1656](https://dev.azure.com/fluidframework/internal/_workitems/edit/1656)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Each example is a standalone application that defines a Fluid Container and load
 
 ## Getting Started
 
-To get started, clone the repository and navigate to one of the example folders. From the example root, you can install and start the example.
+To get started, clone the repository and navigate to the examples folders. Navigate to any of the example in this folder and from the example root, you can install and start the example.
```
Comment (ssimic2) [inline README.md:9]: Do you want to call out that you can npm install/build from the root?

File: build.yml [hunk 2]
```diff
@@ -7,79 +7,79 @@ pool:
...
+
+  - task: UseNode@1
+    displayName: Use Node 12.x
     inputs:
-      testResultsFormat: 'JUnit'
-      testResultsFiles: '**/*junit-report.xml'
-      searchFolder: ${{ parameter.Value }}/nyc
-      mergeTestResults: true
-    condition: succeededOrFailed()
+      version: 12.x
```
Comment (ssimic2) [inline build.yml:51]: Any reason we cannot move to 14?

File: build.yml [hunk 3]
```diff
@@ -7,79 +7,79 @@ pool:
...
+        inputs:
+          command: "custom"
+          workingDir: ${{ parameter.Value }}
+          customCommand: "run lint"
+      - task: Npm@1
+        displayName: Test - ${{ parameter.Value }}
+        inputs:
+          command: "custom"
+          workingDir: ${{ parameter.Value }}
+          customCommand: "run test:report"
```
Comment (ssimic2) [inline build.yml:77]: Does this step run headless puppeteer?

File: build.yml [hunk 4]
```diff
@@ -7,79 +7,79 @@ pool:
...
-- name: node-demo
-  type: string
-  default: node-demo
-- name: teams-fluid-hello-world
-  type: string
-  default: teams-fluid-hello-world
-- name: audience-demo
-  type: string
-  default: audience-demo
+  - name: brainstorm
```
Comment (ssimic2) [inline build.yml:10]: Why the extra space?

**Comment by ssimic2** [comment]
Could you please add description and scope for this PR.

File: README.md (hunk 1 above)
Comment (sonalivdeshpande) [reply README.md]: I have updated the README to run install/build from the root

File: build.yml (hunk 2 above)
Comment (sonalivdeshpande) [reply build.yml:51]: I will make this change as a separate PR :)

File: build.yml (hunk 3 above)
Comment (sonalivdeshpande) [reply build.yml:77]: Currently it does not as the test are not setup yet

File: build.yml (hunk 2 above)
Comment (sonalivdeshpande) [reply build.yml:57]: Updated the yml file

**Comment by tylerbutler** [comment]
One of the build failures is this:

```
Module not found: Error: Can't resolve 'assert' in '/home/vsts/work/1/s/examples/brainstorm/node_modules/@fluidframework/test-runtime-utils/dist'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
```

That should be resolved in the most recent 1.x client release and the latest azure-client.

**Review by tylerbutler — DISMISSED** [review]

**Comment by sonalivdeshpande** [comment]
The angular-demo is not moved to examples folder. There are some build failures seen. Moving this as part of a separate PR.

**Review by tylerbutler — APPROVED** [review]

**Pull request merged**